src/main.ml

```diff
@@ -127,6 +127,28 @@ let output_result output_printer source_path source_content result =
     in
     output_printer (Printer.Replacements { source_path; replacements; result; source_content })
 
+let select_matcher custom_matcher override_matcher configuration =
+  if Option.is_some custom_matcher then
+    Yojson.Safe.from_file (Option.value_exn custom_matcher)
+    |> Matchers.Syntax_config.of_yojson
+    |> function
+    | Ok c -> Matchers.create c
+    | Error error ->
+      Format.eprintf "%s@." error;
+      exit 1
+  else
+    if Option.is_some override_matcher then
```

Comment:
nit:the `else if` can go one line and `ocp-indent` should take care of reindenting so it becomes only one level deep.

Reply:
I actually fixed that but forgot to push the commit :)